server/monitor-types/zabbix-trigger.js

```diff
@@ -0,0 +1,64 @@
+const { MonitorType } = require("./monitor-type");
+const { UP, DOWN } = require("../../src/util");
+const axios = require("axios");
+
+class ZabbixTriggerMonitorType extends MonitorType {
+
+    name = "zabbix-trigger";
+
+    /**
+     * @inheritdoc
+     */
+    async check(monitor, heartbeat, _server) {
+
+        // Make Request
+        const options = {
+            method: "post",
+            url: monitor.zabbixInstanceUrl,
+            data: {
+                "jsonrpc": "2.0",
+                "method": "trigger.get",
+                "params": {
+                    "triggerids": monitor.zabbixTriggerId,
+                    "output": "extend",
+                    "selectFunctions": "extend",
+                },
+                // Will be deprecated, but Bearer Auth is currently not working
```

Comment:
What are you meaning when you say "not working"?

Their docs state that Bearer Auth is the preferred option..
https://www.zabbix.com/documentation/current/en/manual/api#by-authorization-header